live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -76,3 +76,8 @@
   delay: 30
   register: result
   until: result is succeeded
+
+#
+# CRA on Linux requires that the engine UUID is stored in a file
+#
+- shell: dmidecode -s system-uuid | awk '{print tolower($0)}' > /etc/engine-uuid
```

Comment:
This would not work as intended as it will be executed on the build system and so will hardcode that engine-uuid into the install and upgrade image. For migration we would need `/etc/engine-uuid` to be generated dynamically on the customer's system.

Reply:
Right. I'm undoing this part of the change and replacing it with http://reviews.delphix.com/r/50879/ which gets rid of the dependency of CRA on that file altogether.